autocti/extract/one_d/abstract.py

```diff
@@ -137,7 +138,10 @@ def stacked_array_1d_from(
 
         return aa.Array1D(
             values=np.asarray(stacked_array_1d.data),
-            mask=sum(mask_1d_list) == len(mask_1d_list),
+            mask=aa.Mask1D(
+                sum(mask_1d_list) == len(mask_1d_list),
+                pixel_scales=mask_1d_list[0].pixel_scales,
+            ),

```

Comment:
Masks aren't arrays anymore so this had to be cast back explicitly. I might try to make it implicit again

Reply:
Actually I'm not totally sure what the point of this operation is